### DISCUSSION TITLE
Method to flush queue

### DISCUSSION THREAD

**Discussion by dalleggio** [cocotb/cocotb#2944] [Q&A]
I could not find a method to flush a queue in the docs.
Please let me know if there is and I missed it.  Otherwise, it would be very useful to have such a method in case the queue is not empty at the start of the next test.

* the cocotb version 1.6.2
* the operating system Kubuntu 64
* the simulator and version Modelsim De 2020.4
* the Python version 3.7.1, Miniconda

**Comment by eric-wieser** [comment]
You might want to consider just creating a fresh queue for each test. The best way to ensure state doesn't leak from one test to another is to not share the state in the first place.
